addons/payment/tests/test_payment_token.py

```diff
@@ -31,6 +31,11 @@ def test_cannot_assign_token_to_public_partner(self):
     def test_token_cannot_be_unarchived(self):
         """ Test that unarchiving disabled tokens is forbidden. """
```

Comment:
The method name and its docstring should be updated.
```suggestion
    def test_unarchiving_token_requires_active_provider(self):
        """ Test that unarchiving disabled tokens is forbidden if the provider is disabled. """
```